mock_slack_api/slack_server.ts

```diff
@@ -0,0 +1,89 @@
+import bodyParser from 'body-parser';
+import * as dotenv from 'dotenv';
+import express, { RequestHandler } from 'express';
+import { logRequest } from '../src/app.js';
+import Channels from './mock_data/channels.js';
+import Users from './mock_data/user_list.js';
+import { postEphemeral, sendChallenge, sendEvent } from './slack_points.js';
+
+dotenv.config();
+
+const {
+  MOCK_SLACK_PORT: port = '5010',
```

Comment:
Maybe we could extract this and other configs to a new config.ts file where the dotenv is initialized and then export the config object so we can just import the config object and use whatever variable we need?

Reply:
Sounds good to me, will implement.